plugin/pkg/scheduler/core/scheduling_queue_test.go

```diff
@@ -87,12 +88,16 @@ func TestPriorityQueue_Add(t *testing.T) {
 
 func TestPriorityQueue_Pop(t *testing.T) {
 	q := NewPriorityQueue()
+	wg := sync.WaitGroup{}
+	wg.Add(1)
 	go func() {
```

Comment:
This goroutine may not get chance to run.
When `q.Add(...)` returns, this test is done, it may exit before q.Pop() finishes.